### PR TITLE
Forget translation

### PR DESCRIPTION
**Pull request by johannesschobel**
added the following methods
forgetTranslations()
forgetTranslation(array)
syncTranslations(array)
and updated the readme

### CODE REVIEW

**Comment by johannesschobel** [comment]
Hey @dimsav ,
I have added the methods discussed in #254 . Maybe you can take a look at the pull request so we can discuss this issue?
Greetings and Cheers,
Johannes

**Comment by johannesschobel** [comment]
Just added further information regarding the ```$this->load("translations");``` call:
[Model Relationship not updated after deleting related models](http://laravel.io/forum/12-17-2014-model-relationship-not-updated-after-deleting-related-models)

File: src/Translatable/Translatable.php [hunk 1]
```diff
@@ -624,4 +624,57 @@ protected function locale()
         return App::make('config')->get('translatable.locale')
             ?: App::make('translator')->getLocale();
     }
+
+    /**
+     * Removes all translations for this model.
+     */
+    public function forgetTranslations()
+    {
+        $modelTranslation = $this->getTranslationModelName();
```
Comment (dimsav) [inline src/Translatable/Translatable.php]: Please change these 2 lines to

```php
$this->translations()->delete();
```

File: src/Translatable/Translatable.php [hunk 2]
```diff
@@ -624,4 +624,57 @@ protected function locale()
         return App::make('config')->get('translatable.locale')
             ?: App::make('translator')->getLocale();
     }
+
+    /**
+     * Removes all translations for this model.
+     */
+    public function forgetTranslations()
```
Comment (dimsav) [inline src/Translatable/Translatable.php]: Please rename this to `deleteTranslations()`

**Comment by johannesschobel** [comment]
done

**Comment by dimsav** [comment]
@johannesschobel are you at the laravel slack channel?

**Comment by dimsav** [comment]
If not please join here: https://laravelgreece.slack.com

**Comment by johannesschobel** [comment]
How do I join? I have a slack team, but i am not able to join your slack team?!

**Comment by dimsav** [comment]
hmm true. Sent you an invite to johannes.schobel(at)uni-ulm.de

**Pull request merged**